packages/mds-cache/index.ts

```diff
@@ -143,17 +143,17 @@ async function getEventsInBBox(bbox: BoundingBox) {
   return client.georadiusAsync('locations', lng, lat, radius, 'm')
 }
 
-async function hreads(suffixes: string[], device_ids: UUID[]): Promise<CachedItem[]> {
+async function hreads(suffixes: string[], ids: UUID[], prefix = 'device'): Promise<CachedItem[]> {
   if (suffixes === undefined) {
     throw new Error('hreads: no suffixes')
   }
-  if (device_ids === undefined) {
+  if (ids === undefined) {
     throw new Error('hreads: no device_ids')
```

Comment:
This error message is perhaps a little misleading now.

Reply:
That makes sense, maybe we should just say `no ids provided`